### PR TITLE
Add maintenance task

### PR DESCRIPTION
**Pull request by kelceydamage**
Adding a task to increase swapfile size on production servers.

### CODE REVIEW

File: ma/plans/2020-06-18.md [hunk 1]
```diff
@@ -10,42 +10,5 @@ Bastillion aws instances named bastillion1.int.flyzipline.com bastillion2.int.fl
...
-View the audit logs of that connection.
-
-Assuming tests pass, do the following:
-
-Notify stakeholders that maintenance on production bastillion will occur during the scheduled window.
-
-#### Rollback Notes
-Stop the bastillion service, move the upgraded release out of the way and extract the backup tarball. Restart the bastillion service.
-
+* [Upgrade bastillion and remove snap support](https://github.com/kelceydamage/soporific/blob/patch-1/ma/plans/upgrade-bastillion.md)
```
[the remark this answers is not in the document]
Comment (kelceydamage) [reply ma/plans/2020-06-18.md:9]: yes

File: ma/plans/MAINTENANCE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+### <NAME OF CHANGE>
```
[the remark this answers is not in the document]
Comment (kelceydamage) [reply ma/plans/MAINTENANCE_TEMPLATE.md]: Outdated. There is a scope section

File: ma/plans/increase-swapfile-size.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+### Increase swap files from 4 GB to 8GB.
+We have found that our swapfile sizes are too low and can lead to extreneous alerts.
```
[the remark this answers is not in the document]
Comment (kelceydamage) [reply ma/plans/increase-swapfile-size.md:5]: We don't have 6 months of Prometheus data. We have an alert with swap on some servers at 99%. The team decided the current best course of action was to increase swap, considering we have 32GB system ram, swapiness=60 and only 4 GB swap.

[the remark this answers is not in the document]
Comment (kelceydamage) [reply ma/plans/increase-swapfile-size.md:5]: The cause is `not enough ram` and ram cost too much $$$ :)

File: ma/plans/increase-swapfile-size.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+There is sufficient headroom on the systems to avoid `swapoff failed: Cannot allocate memory`, when disabling swap in order to modify it.
+
+#### Stakeholders
+Team-engineering slack channel members.
+
+#### Procedure
+The proceedure to increase the amout of available swap is as follows:
+
+On all servers:
+1. Create the new swapfile: `sudo dd if=/dev/zero of=/swapfile bs=1M count=8192`
```
[the remark this answers is not in the document]
Comment (kelceydamage) [reply ma/plans/increase-swapfile-size.md:17]: Currently no. As per monitoring telemetry the root partitions should have sufficient space. If not we can prune stale docker images, and old logs.

File: ma/plans/increase-swapfile-size.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+2. Disable swap with: `swapoff -a`
+3. Set the new swapfile as swap: `mkswap /swapfile`
+4. re-enable swap: `swapon /swapfile`
+
+#### Testing
+Testing was performed on `nest0-server-a` and `nest0-server-b`
+
+#### Rollback Notes
+To roll back the change repeat the process and in the `dd` stage set the count to the original size (4096 for 4GB swap, etc).
```
[the remark this answers is not in the document]
Comment (kelceydamage) [reply ma/plans/increase-swapfile-size.md:26]: I made a node about removing swap when no memory available. I don;t know what a `connect to oob` safety step is.

File: ma/plans/2020-06-18.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kelceydamage) [reply ma/plans/2020-06-18.md]: Resolved in commit.

File: ma/plans/increase-swapfile-size.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (flopsec) [reply ma/plans/increase-swapfile-size.md:26]: Oh I just meant having ipmi console access sorted before we proceed (assuming physical)

**Pull request merged**